js/apps/admin-ui/src/components/dynamic/UrlComponent.tsx

```diff
@@ -0,0 +1,44 @@
+import { FormGroup } from "@patternfly/react-core";
+import { useTranslation } from "react-i18next";
+
+import { HelpItem } from "ui-shared";
+import { Controller, useFormContext } from "react-hook-form";
+import type { ComponentProps } from "./components";
+import { FormattedLink } from "../external-link/FormattedLink";
+import { convertToName } from "./DynamicComponents";
+
+import "./url-component.css";
+
+export const UrlComponent = ({
+  name,
+  label,
+  helpText,
+  defaultValue
+}: ComponentProps) => {
+  const { t } = useTranslation();
+  const { control } = useFormContext();
+  
+  return (
+    
+    <FormGroup
+      label={t(label!)}
+      fieldId={name!}
+      labelIcon={<HelpItem helpText={t(helpText!)} fieldLabelId={`${label}`} />}
+      isRequired={false}
```

Comment:
This is the default, so you can remove this line

Reply:
Removed